.github/workflows/codeql-analysis.yml

```diff
@@ -21,25 +21,25 @@ defaults:
 
 jobs:
 
-  changes:
-    name: Check changes
+  conditional:
+    name: Check conditional workflows and jobs
     if: github.event_name != 'schedule' || github.repository == 'keycloak/keycloak'
     runs-on: ubuntu-latest
     outputs:
-      java: ${{ steps.changes.outputs.java }}
-      themes: ${{ steps.changes.outputs.themes }}
-      js-adapter: ${{ steps.changes.outputs.js-adapter }}
+      java: ${{ steps.conditional.outputs.codeql-java }}
+      themes: ${{ steps.conditional.outputs.codeql-themes }}
+      js-adapter: ${{ steps.conditional.outputs.codeql-js_adapter }}
     steps:
       - uses: actions/checkout@v3
```

Comment:
Do we need to check out the repository for this part to work? If so perhaps it makes more sense to add the checkout of the repository to the `conditional` workflow itself? (same for the operator workflow)

Reply:
Yes, composite actions run directly from the checkout, and won't work if you don't do the checkout. It's very fast though, so not really a big problem.